pkg/operator/encryption/controllers/condition_controller.go

```diff
@@ -63,8 +63,21 @@ func NewConditionController(
 	).ResyncEvery(time.Minute).WithSync(c.sync).ToController("EncryptionConditionController", eventRecorder.WithComponentSuffix("encryption-condition-controller"))
 }
 
-func (c *conditionController) sync(ctx context.Context, syncContext factory.SyncContext) error {
+func (c *conditionController) sync(_ context.Context, _ factory.SyncContext) (err error) {
+	condition := &operatorv1.OperatorCondition{Type: "Encrypted", Status: operatorv1.ConditionFalse}
```

Comment:
keep name `cond` and the patch gets much smaller.

Reply:
done